### ISSUE TITLE
Matlab Executable 404

### ISSUE THREAD

**Issue by TescaF** [lubiluk/skill_transfer#5]
The Matlab executable listed in the Readme links to a 404'd page:
https://github.com/pauloabelha/enzymes/blob/master/Bremen/edge_detector/for_redistribution/edge_detector_installer.install

Do you have an updated link?

**Comment by lubiluk** [comment]
Good catch. Seems that my Readme is outdated. I think we stopped using edge_detector in favor of `get_tool_info` and `get_target_obj_info`. Their executables can be found here:

https://github.com/pauloabelha/enzymes/tree/master/IROS2018/ROS_CV_node/get_tool_info
https://github.com/pauloabelha/enzymes/tree/master/IROS2018/ROS_CV_node/get_target_obj_info/get_target_obj_info/for_redistribution

You've got to concatenate those *split* files.
